### DISCUSSION TITLE
CLI: Support passing a token through an env variable

### DISCUSSION THREAD

**Discussion by jld-adriano** [vercel/community#7401] [Idea]
I manage my own tokens, and would love to be able to just pass the token through the env such as 

`VERCEL_TOKEN=xxx vercel whomai`

This would make the usage transparent, while now I have to alias all vercel commands to include 

`alias vercel='vercel --token=${VERCEL_TOKEN}'`

**Comment by pawlean** [comment]
Thanks for the suggestion, @jld-adriano! Passed on internally.

[a comment by an automated account is not shown]
